tests/components/airzone/test_config_flow.py

```diff
@@ -31,12 +28,12 @@
     USER_INPUT,
 )
 
-from tests.common import MockConfigEntry
+from tests.common import MockConfigEntry, MockDhcpServiceInfo
 
-DHCP_SERVICE_INFO = DhcpServiceInfo(
+DHCP_SERVICE_INFO = MockDhcpServiceInfo(
     hostname="airzone",
     ip="192.168.1.100",
-    macaddress=dr.format_mac("E84F25000000").replace(":", ""),
+    macaddress="E84F25000000",
```

Comment:
The MAC here was correctly formatted and stripped of colons, but now the Mock class does it automatically